oap-ape/ape-java/ape-spark/src/main/java/org/apache/spark/sql/execution/vectorized/RemoteColumnVector.java

```diff
@@ -177,19 +179,8 @@ public ColumnVector getChild(int i) {
     throw new UnsupportedOperationException("Not support yet");
   }
 
-  private void reset() {
-    // Only element length buffer is sliced from batch response.
-    // Data buffer and null buffer will be released by response itself.
-    if (elementLengthBuf != null) {
-      elementLengthBuf.release();
-      elementLengthBuf = null;
-    }
-  }
-
   @Override
-  public void close()  {
-    reset();
-  }
+  public void close()  {}
```

Comment:
okay to do nothing here? Can you add some comments to illustrate?

Reply:
Since the buffers in response are released in nextBatch, I just add code to set the bytebufs to null in new commit.